backend/src/services/payments/payment-customer.service.ts

```
@@ -0,0 +1,510 @@
+  private buildBulkUpsertCustomerParams(
+    environment: StripeEnvironment,
+    customers: StripeCustomerLike[],
+    syncedAt: Date
+  ): Array<
+    | StripeEnvironment
+    | string
+    | boolean
+    | Record<string, string>
+    | StripeCustomerLike
+    | Date
+    | null
```

ESLint

[error] Replace `|·StripeEnvironment⏎····|·string⏎····|·boolean⏎····|·Record<string,·string>⏎····|·StripeCustomerLike⏎····|·Date⏎···` with `StripeEnvironment·|·string·|·boolean·|·Record<string,·string>·|·StripeCustomerLike·|·Date`

(prettier/prettier)